packages/itwinui-css/package.json

```diff
@@ -29,30 +29,34 @@
   ],
   "dependencies": {},
   "devDependencies": {
+    "@itwin/itwinui-icons-elements": "0.2.0",
     "autoprefixer": "^10.4.1",
     "backstopjs": "~6.0.4",
     "html-minifier": "^4.0.0",
+    "npm-run-all": "^4.1.5",
     "postcss": "^8.4.5",
     "postcss-discard-comments": "^5.0.1",
     "postcss-scss": "^4.0.3",
     "sass-embedded": "^1.49.9",
+    "serve": "^14.0.1",
```

Comment:
Why it is needed? Can't you use preview from vite?

Reply:
preview wasn't showing anything for me
![ ](https://user-images.githubusercontent.com/9084735/179760534-55c1a394-fe91-4a7f-8c99-3dd90af337ca.png)

Reply:
Have you tried looking into that?

Reply:
I don't see any logs or anywhere so I was very confused and decided to just go with `serve`.

Reply:
Maybe ok, just not to waste time for this minor stuff.